packages/react/src/blocks/Image.tsx

```diff
@@ -99,65 +99,71 @@ class ImageComponent extends React.Component<any> {
         {value => {
           const amp = value.ampMode
           const Tag: 'img' = amp ? ('amp-img' as any) : 'img'
+
+          const ImageContents = (!lazy || this.state.load) && (
+            <Tag
+              {...(amp
+                ? ({
+                    layout: 'responsive',
+                    height:
+                      this.props.height ||
+                      (aspectRatio ? Math.round(aspectRatio * 1000) : undefined),
+                    width:
+                      this.props.width ||
+                      (aspectRatio ? Math.round(1000 / aspectRatio) : undefined)
+                  } as any)
+                : null)}
+              alt={this.props.altText}
+              key={
+                Builder.isEditing
+                  ? (typeof this.props.image === 'string' &&
+                      this.props.image.split('?')[0]) ||
+                    undefined
+                  : undefined
+              }
+              // height={
+              //   this.props.height || (aspectRatio ? Math.round(aspectRatio * 1000) : undefined)
+              // }
+              // width={
+              //   this.props.width || (aspectRatio ? Math.round(1000 / aspectRatio) : undefined)
+              // }
+              role={!this.props.altText ? 'presentation' : undefined}
+              css={{
+                objectFit: this.props.backgroundSize,
+                objectPosition: this.props.backgroundPosition,
+                ...(aspectRatio && {
+                  position: 'absolute',
+                  height: '100%',
+                  width: '100%',
+                  left: 0,
+                  top: 0
+                }),
+                ...(amp && {
+                  ['& img']: {
+                    objectFit: this.props.backgroundSize,
+                    OObjectPosition: this.props.backgroundPosition
```

Comment:
oh this is actually a typo from the original code - should be `objectPosition: ...`